Eos/Fuego/EOS.H

```diff
@@ -264,7 +264,8 @@ AMREX_FORCE_INLINE
 void
 EY2G(amrex::Real E, amrex::Real Y[NUM_SPECIES], amrex::Real& G)
 {
-  amrex::Real wbar, Cv, T, Cvx;
+  amrex::Real wbar, Cv, Cvx;
+  amrex::Real T = 300.;
```

Comment:
We typically have a good guess for T laying around, and that guess could be passed to this function through another argument.  This seems to be different than the other functions for some reason, no?  Is it worth considering changing this to be EYT2G and passing T?